src/chainparams.cpp

```diff
@@ -298,3 +337,29 @@ bool SelectParamsFromCommandLine()
     SelectParams(network);
     return true;
 }
+
+
+// Block height must be >0 and <=last founders reward block height
+// Index variable i ranges from 0 - (vFoundersRewardAddress.size()-1)
+std::string CChainParams::GetFoundersRewardAddress(int nHeight) const
+{
+    int maxHeight = consensus.GetLastFoundersRewardBlockHeight();
+    assert(nHeight>0 && nHeight<=maxHeight);
+
+    size_t i = (size_t)floor((double(nHeight-1)/maxHeight)*vFoundersRewardAddress.size());
```

Comment:
This differs from the spec. The spec requires an integer division of ``nHeight`` by FounderAddressChangeInterval := ceiling((SlowStartShift + HalvingInterval) / NumFounderAddresses. The above code has different rounding behaviour.